extensions/amp-analytics/0.1/events.js

```diff
@@ -62,72 +67,72 @@ const ALLOWED_FOR_ALL_ROOT_TYPES = ['ampdoc', 'embed'];
  *   }>}
  */
 const TRACKER_TYPE = Object.freeze({
-  'click': {
-    name: 'click',
+  [AnalyticsEventType.CLICK]: {
```

Comment:
Didn't know you can use `[]` to retrieve the value : )

Reply:
realized `ComputedPropertyName` is a ES6 feature. so there won't be any bytes saving in our current binary as babel will convert them back to raw string.

Reply:
that said, we will only have saving once we start to deliver ES6 code to browser.

Reply:
TBH I don't even think there will be saving even we deliver ES6 code. I could be wrong, curious ifthere a way to test this out

Reply:
@erwinmombay and @jridgewell 
I am curious about two things
1. What's the plan to deliver ES6 code? 
2. Do you think that after brotli or gzip compression, such change (change string to property value) will bring size save? And what's the best way for me to find out locally. 
Thank you!

Reply:
> What's the plan to deliver ES6 code?

We're working on it with `module`/`nomodule`

> Do you think that after brotli or gzip compression, such change (change string to property value) will bring size save? And what's the best way for me to find out locally.

No idea if this will be beneficial to code size. But in this case, I think the improvement to readability is worth it alone.

Only [brotli matters](https://caniuse.com/#feat=brotli). You'd have to build locally, then run `brotli --best FILE | wc -c`

Reply:
Thank you @jridgewell 
Another question: How can I build locally in ES6? Not very familiar with the build system. I assume I can ran `gulp dist`?